Translations/translation_RO.json

```diff
@@ -0,0 +1,214 @@
+{
+  "languageCode": "RO",
+  "languageLocalName": "Română",
+  "fonts": ["ascii_basic", "latin_extended"],
+  "tempUnitFahrenheit": true,
+  "messages": {
+    "SettingsCalibrationDone": "Calibration done!",
+    "SettingsCalibrationWarning": "Vă rugăm să vă asigurati că vârful este la temperatura camerei, înainte de a continua!",
+    "SettingsResetWarning": "Sigur doriti să restaurati la setările implicite?",
+    "UVLOWarningString": "DC LOW",
+    "UndervoltageString": "Sub tensiune",
+    "InputVoltageString": "Intrare V: ",
+    "WarningTipTempString": "Tip temp: ",
+    "BadTipString": "BAD TIP",
+    "SleepingSimpleString": "Zzzz",
+    "SleepingAdvancedString": "Adormit...",
+    "WarningSimpleString": "HOT!",
+    "WarningAdvancedString": "!!! HOT TIP !!!",
+    "SleepingTipAdvancedString": "Tip:",
+    "IdleTipString": "Tip:",
+    "IdleSetString": " Set:",
+    "TipDisconnectedString": "FĂRĂ SFAT",
+    "SolderingAdvancedPowerPrompt": "Putere: ",
+    "OffString": "Off",
+    "YourGainMessage": "Your gain:"
```

Comment:
@sandmanRO

Reply:
"TipDisconnectedString": "FARA VARF" or "VARF LIPSA"
"OffString": "Nu",
"YourGainMessage": "Coeficient:"
(power is ok)